README-translated/README-Hindi.md

```diff
@@ -0,0 +1,127 @@
+| **समाचार** | [![Twitter Follow](https://img.shields.io/twitter/follow/turingway?style=social)](https://twitter.com/turingway) [![Join our tinyletter mailing list](https://img.shields.io/badge/receive-our%20newsletter%20❤%EF%B8%8F-blueviolet.svg)](https://tinyletter.com/TuringWay)  [![YouTube Channel Views](https://img.shields.io/youtube/channel/views/UCPDxZv5BMzAw0mPobCbMNuA?style=social)](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)  |
+| **चैट** | [![Join Slack](https://img.shields.io/badge/Chat-on%20Slack-ff69b4)](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw) [![GitHub issues](https://img.shields.io/github/issues/alan-turing-institute/the-turing-way)](https://github.com/alan-turing-institute/the-turing-way/issues) [![GitHub pull requests](https://img.shields.io/github/issues-pr/alan-turing-institute/the-turing-way)](https://github.com/alan-turing-institute/the-turing-way/pulls) |
+| **संकलन** | [![Netlify build](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster) [![Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
+| **मुलाकात** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |
+
+*"द ट्यूरिंग वे" परियोजना के गीटहब (GitHub) भंडार में आपका स्वागत है। यहां सभी परियोजना घटकों का विकास, समीक्षा और रख-रखाव किया जाता है।.*
```

Comment:
```suggestion
*"द ट्यूरिंग वे" परियोजना के गीटहब (GitHub) भंडार में आपका स्वागत है। यहां सभी परियोजना घटकों का विकास, समीक्षा और रख-रखाव किया जाता है।.*
```
should be .....का विकास, unki समीक्षा और unka रख-रखाव किया जाता है।
Alt suggestion: …. pariyojna ghatakon ki sameeksha, unka vikas aur rakh-rakhav kiya jata hai.

....Because the gender of the words change alternatively, it needs the respective unka /ki in front of it to be better read. It is however correct as a direct translation. If you would prefer a semantic translation however, I would suggest  ‘ pariyojna ghatakon ki sameeksha, unka vikas aur rakh-rakhav kiya jata hai.’ To avoid writing too many unka/unki or having the awkward pause while reading if you’re not adding the unka/ki.